cloud-controller-manager/do/loadbalancers_test.go

```diff
@@ -2184,327 +2184,63 @@ func Test_buildHealthCheck(t *testing.T) {
 		errMsgPrefix string
 	}{
 		{
-			name: "tcp health check",
```

Comment:
Should we bring some/all of the tests back now that we still support it (through the fallback annotation)?

Reply:
I have added them back